main/guides/getting-started/contract-rpc.md

```diff
@@ -47,6 +47,24 @@ message MsgWalletSpendAction {
 
 ## Querying VStorage
 
+::: tip VStorage as Contract State Query Mechanism
+Unlike Ethereum and other blockchains that provide [readonly or view query APIs](https://docs.soliditylang.org/en/latest/contracts.html#view-functions) directly in the smart contracts,
+the Agoric chain does not offer such direct readonly APIs for querying contract state. Instead, the Agoric
+platform achieves this functionality through a powerful feature called VStorage.
+
+Smart contracts on the Agoric chain can write data to VStorage, which acts as a publicly accessible
+storage layer. This data is available for anyone to query, similar to the functionality of a getter
+API on other blockchains. It is completely cost-free, meaning developers and users can fetch data
+from VStorage without incurring any fees.
+
+For instace, if a smart contract manages educational records, it can publish any relevant information
+to VStorage through a `chainStorage` API, and anyone can query this data to verify the authenticity of
+the records. This approach is illustrated in the [dapp-ed-cert.
```

Comment:
dapp-ed-cert link seems incomplete.

Reply:
It would be nice to have a small snippet of code inline here.

Reply:
Regarding the snippet, I am not sure at this point. ATM `dapp-ed-cert` using `E()`, as in:

```js
const edCertNode = E(certificateDataRoot).makeChildNode(edCert.certificateId);
await E(edCertNode).setValue(JSON.stringify(edCert));
```
I would rather add the snippet later after I have fixed it.